spec/support/macros/deprecation.rb

```diff
@@ -13,6 +13,8 @@ def silence_deprecation(example)
   config.include SilenceDeprecation
 
   config.around :example, silence_deprecation: true do |example|
-    silence_deprecation(example)
+    with_silenced_output do
```

Comment:
I am not a fan of this `with_silenced_output` method. This is not longer just silencing the deprecation, it is silencing any output, which would make things hard to debug if anything went wrong.

Instead, I think we can just update the `silence_deprecation` to set `FactoryBot::DEPRECATOR.silenced` instead of `ActiveSupport::Deprecation.silenced`.

Reply:
We might also consider renaming `FactoryBot::DEPRECATOR` to `FactoryBot::Deprecation` so it better matches `ActiveSupport::Deprecation`.